azure-quantum/requirements-qiskit.txt

```diff
@@ -1,5 +1,5 @@
-qiskit-ionq>=0.3.3,<1.0
-qiskit-qir>=0.4,<0.5
-qiskit-terra>=0.46,<0.47
+qiskit-ionq==0.5.0
+qiskit-qir==0.5.0rc1
```

Comment:
We could now specify the release version of `qiskit-qir>=0.5,<0.6` as it got just released recently
https://pypi.org/project/qiskit-qir/0.5.0/